sos/report/plugins/microshift_ovn.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2023 Pablo Acevedo <pacevedo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class MicroShiftOVN(Plugin, RedHatPlugin):
+    """This plugin is used to collect MicroShift 4.x OVN logs.
+    """
+    short_desc = 'MicroShift OVN'
+    plugin_name = "microshift_ovn"
+    profiles = ('microshift',)
```

Comment:
Should not be there some enabling condition like `packages=..` or `services=..` or `containers=..` ? Currently the plugin is enabled on either system, every time.

Reply:
You are right, it was a leftover. Fixed now.